qt/python/mantidqt/widgets/plotconfigdialog/axestabwidget/view.py

```diff
@@ -55,10 +55,10 @@ def set_title(self, title):
 
     # X-Axis getters
     def get_xlower_limit(self):
-        return self.xlower_limit_spin_box.value()
+        return self.xlower_limit_line_edit.text()
```

Comment:
Should we convert to a `float` here? The `QDoubleValidator` should guarantee this is the case. The presenter can just assume it gets a float back then (similarly for the other limits)